src/tools-and-bots/crater.md

```diff
@@ -0,0 +1,17 @@
+# `@craterbot`
+
+**Status:** Stub
+
+[Crater](https://github.com/rust-lang/crater) is a tool that can test PRs against a public subset of the Rust ecosystem to estimate the scale of potential breakage.
+
+You can kick off a crater run with:
+
+```
+@bors try @craterbot check
```

Comment:
Does this work? I was under the impression that unlike rust-timer, craterbot doesn't wait for a `bors try` and can only be started after the `bors try` build has finished.

Reply:
I think you're right and it has to be separate. It might be nice to extend craterbot to do this, though.

Reply:
😢